agent/fw_wordpress.c

```diff
@@ -551,6 +551,22 @@ NR_PHP_WRAPPER(nr_wordpress_apply_filters) {
 }
 NR_PHP_WRAPPER_END
 
+void nr_wordpress_version() {
+  char* string = "$GLOBALS['wp_version'];";
```

Comment:
Same `const` suggestion as https://github.com/newrelic/newrelic-php-agent/pull/775/files#r1440962256

Reply:
After discussion, this will not be added. Same reasoning as: https://github.com/newrelic/newrelic-php-agent/pull/775#discussion_r1442174308